Source/UETorch/Private/UETorch.cpp

```diff
@@ -118,6 +118,10 @@ typedef struct {
  */
 extern "C" void GetViewportSize(IntSize* r)
 {
+  if(GEngine->GameViewport->Viewport == NULL){
```

Comment:
Are you sure that GEngine->GameViewport is never NULL? Maybe check that as well.